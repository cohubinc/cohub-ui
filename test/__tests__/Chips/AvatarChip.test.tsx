import React from "react";
import { mount } from "enzyme";

import { Chip } from "dist";

it("renders internal Link without crashing", async () => {
  const wrapper = mount(<Chip.Avatar name="Elliott R." />);

  expect(wrapper).toMatchSnapshot();
});
