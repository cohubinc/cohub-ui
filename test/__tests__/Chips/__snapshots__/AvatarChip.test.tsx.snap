// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders internal Link without crashing 1`] = `
<AvatarChip
  name="Elliott R."
>
  <Chip
    backgroundColor="hsl(0, 0%, 96%)"
    className="mb-1"
    dark={true}
    size={12}
    style={
      Object {
        "marginRight": "4px",
        "paddingBottom": "4px",
        "paddingTop": "4px",
      }
    }
  >
    <div
      className="CohubChip  mb-1"
      style={
        Object {
          "backgroundColor": "hsl(0, 0%, 96%)",
          "borderRadius": "361px",
          "display": "inline-block",
          "marginRight": "4px",
          "padding": "4.8px 12px",
          "paddingBottom": "4px",
          "paddingTop": "4px",
        }
      }
    >
      <div
        className="flex justify-center items-center h-100"
        style={
          Object {
            "cursor": "inherit",
          }
        }
      >
        <Component>
          <span
            className=""
            data-qa="text"
            style={
              Object {
                "color": "hsl(0, 2%, 22%)",
                "fontFamily": "Inter",
                "fontSize": 12,
                "fontWeight": 400,
                "textAlign": "initial",
                "textTransform": "initial",
              }
            }
          >
            <div
              className="flex items-center"
            >
              <Avatar
                className="mr-05"
                size={20}
              >
                <div
                  className="mr-05"
                >
                  <Component
                    size={20}
                  >
                    <DefaultIcon
                      name="user"
                      size={20}
                    >
                      <User
                        name="user"
                        size={20}
                      >
                        <IconWrapper
                          name="user"
                          size={20}
                        >
                          <div
                            className="CohubIcon userIcon  "
                            style={
                              Object {
                                "height": 20,
                                "opacity": undefined,
                                "transition": "opacity 300ms ease-in 200ms",
                                "width": 20,
                              }
                            }
                          >
                            <div
                              className="flex justify-center items-center"
                            >
                              <svg
                                height={20}
                                viewBox="0 0 20 20"
                                width={20}
                              >
                                <mask
                                  height={20}
                                  id="050eb4cb-74cf-fad8-c493-b16c0d3be973"
                                  mask-type="alpha"
                                  maskUnits="userSpaceOnUse"
                                  width={20}
                                  x="0"
                                  y="0"
                                >
                                  <circle
                                    cx={10}
                                    cy={10}
                                    fill="#F2F2F2"
                                    r={10}
                                  />
                                </mask>
                                <g
                                  mask="url(#050eb4cb-74cf-fad8-c493-b16c0d3be973)"
                                >
                                  <circle
                                    cx={10}
                                    cy={10}
                                    fill="#F2F2F2"
                                    r={10}
                                  />
                                  <circle
                                    cx={10}
                                    cy={18.33333333348611}
                                    fill="#C4C4C4"
                                    r={6.666666666666667}
                                  />
                                  <circle
                                    cx={10}
                                    cy={7.49999999990625}
                                    fill="#C4C4C4"
                                    r={3.3333333333333335}
                                  />
                                </g>
                              </svg>
                            </div>
                          </div>
                        </IconWrapper>
                      </User>
                    </DefaultIcon>
                  </Component>
                </div>
              </Avatar>
              <Component>
                <span
                  className=""
                  data-qa="text"
                  style={
                    Object {
                      "color": "hsl(0, 2%, 22%)",
                      "fontFamily": "Inter",
                      "fontSize": 12,
                      "fontWeight": 400,
                      "textAlign": "initial",
                      "textTransform": "initial",
                    }
                  }
                >
                  Elliott R.
                </span>
              </Component>
            </div>
          </span>
        </Component>
      </div>
    </div>
  </Chip>
</AvatarChip>
`;
